lib/gds_api/publishing_api_v2.rb

```diff
@@ -315,6 +315,28 @@ def get_content_items(params)
     get_json("#{endpoint}/v2/content#{query}")
   end
 
+  # Returns an Enumerator of content items for the provided
+  # query string parameters.
+  #
+  # @param params [Hash]
+  #
+  # @return [Enumerator] an enumerator of content items
+  #
+  # @see https://github.com/alphagov/publishing-api/blob/master/doc/api.md#get-v2content
+  def get_paged_content_items(params, per_page: 50)
```

Comment:
Given that this is a wrapper around `get_content_items` that provides an enumerator, I wonder if a name like `get_content_items_enumerator` might be clearer?

While the internals has something to do with pagination, compared to the `get_content_items` function, this is the one that involves no direct interaction with the pagination, so keeping that out of the name would be nice I think.